cypress/e2e/notes-100/access-shared-note.cy.js

```diff
@@ -22,7 +22,7 @@ describe('Notes 100 - Access shared note', () => {
   })
   context('Returning user accessing share through the link that contains specific NOTE PREFIX', () => {
     beforeEach(() => {
-      cy.visit('/share/v/p/index.ifc#c:-26.91,28.84,112.47,-22,16.21,-3.48;i:2')
+      cy.visit('/share/v/p/index.ifc#c:-26.91,28.84,112.47,-22,16.21,-3.48;i:126')
       waitForModel()
     })
       it('Panel title to contain NOTE string', () => {
```

Comment:
The following `it`s are just checks, so please collapse.  you have a .only as well

Reply:
I split the its into two, one for the the nav and one for the noteCard.